control-plane/plugin/src/resources/mod.rs

```diff
@@ -104,10 +104,22 @@ pub enum DrainResources {
 /// The types of resources that support the 'label' operation.
 #[derive(clap::Subcommand, Debug)]
 pub enum LabelResources {
-    /// Label node with the given ID.
+    /// Adds or removes a label from the specified node.
```

Comment:
`Adds or removes a label from the specified node.` --> `Adds or removes a label to or from the specified node.`